docs/_articles/en/user-guide/detect-conflicts.md

```diff
@@ -4,7 +4,7 @@ lang: en
 ---
 ## Overview
 
-When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the Org. The conflict detection feature is available for any org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs.
+When you deploy source to your default org, the operation could detect conflicts on deploy to help you avoid accidentally overwriting others’ changes in the Org. The conflict detection feature can be turned on for any org that is not using source tracking, which is most non-scratch orgs such as sandboxes or dev orgs. Note that this feature is always on for scratch orgs and source tracked sandboxes.
```

Comment:
I've seen and written "non-source-tracked", "non-source tracked" and "non source-tracked".  Which one should be settle on? :)

Reply:
Oh, I see.  Here we say non-scratch org.  Ok.  And above you have "non source-tracked".  👍🏽